packages/perseus-editor/src/iframe-content-renderer.tsx

```diff
@@ -115,17 +115,14 @@ class IframeContentRenderer extends React.Component<Props> {
     _lastData: any;
     // @ts-expect-error - TS2564 - Property '_lastHeight' has no initializer and is not definitely assigned in the constructor.
     _lastHeight: number;
-    // @ts-expect-error - TS2564 - Property 'iframeID' has no initializer and is not definitely assigned in the constructor.
-    iframeID: number;
+
+    iframeID = nextIframeID++;
```

Comment:
Would this work? Where is `nextIframeID` defined?

Reply:
It's defined as a `let` at the top level of this file, on line 59.